genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :items
+  attr_reader :id
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
```

Comment:
Both Student 1 and Student 3 agreed upon generating a random id int between 1 and 10_000 (for the sake of lowering the chances of the id int overlapping with other ids).